docs/topics/vs-code-extension-resolving-issues.adoc

```diff
@@ -6,14 +6,12 @@
 [id="vs-code-extension-resolving-issues_{context}"]
 = Resolving issues
 
-You can resolve issues by doing one of the following:
-
-* Using a _Quick Fix_ automatic code replacement
-* Editing the code of a file with a hint
+You can resolve issues by editing the code of a file with a hint
 
 You can keep track of resolved issues by using the *Delete* or *Mark as Complete* options. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
```

Comment:
`You can keep track of resolved issues by using the *Delete* or *Mark as Complete* options. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.`
This needs to be removed, or at least commented out, this functionality is not available in vscode.